packages/backend/src/createRedirect.ts

```diff
@@ -70,23 +71,35 @@ type CreateRedirect = <ReturnType>(params: {
   baseUrl: URL | string;
   signInUrl?: URL | string;
   signUpUrl?: URL | string;
+  sessionStatus?: JwtPayload['sts'];
 }) => {
   redirectToSignIn: RedirectFun<ReturnType>;
   redirectToSignUp: RedirectFun<ReturnType>;
 };
 
 export const createRedirect: CreateRedirect = params => {
-  const { publishableKey, redirectAdapter, signInUrl, signUpUrl, baseUrl } = params;
+  const { publishableKey, redirectAdapter, signInUrl, signUpUrl, baseUrl, sessionStatus } = params;
   const parsedPublishableKey = parsePublishableKey(publishableKey);
   const frontendApi = parsedPublishableKey?.frontendApi;
   const isDevelopment = parsedPublishableKey?.instanceType === 'development';
   const accountsBaseUrl = buildAccountsBaseUrl(frontendApi);
 
+  const redirectToTasks = ({ targetUrl, returnBackUrl }: RedirectToParams & { targetUrl: string | URL }) => {
+    const rootTasksPath = 'tasks';
+    const tasksUrl = new URL(rootTasksPath, targetUrl);
```

Comment:
This logic seems to be broken now, getting an URL constructor error, fixing it now!